docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+services:
+
+  backend:
+    image: python:3.5
+    command: python3 /app/refactor_me.py
+    environment:
+      - PYTHONUNBUFFERED=1
+    volumes:
+      - ./:/app
+
+  flake:
```

Comment:
why not pylint?

Reply:
there are just no reason for that. It does not metter. I have a stong eye to find PEP8 mismatches, but flake, sometimes, helps, when refers to unused imports...